### DISCUSSION TITLE
support generate Qwen model  tokenizer? support load tikToken?

### DISCUSSION THREAD

**Discussion by gaboness** [deepjavalibrary/djl#3218] [Q&A]
now i want to calculate Qwen model input/output tokens length by djl, but Qwen model use qwen.tiktoken not tokenizer.json,  is there a way to load qwen.tiktoken or can translate ?

**Comment by frankfliu** [comment]
Currently DJL use fasttokenizer from [huggingface tokenizer](https://github.com/huggingface/tokenizers). It's possible to implement TikTokenizer in java, would like to contribute?

**Reply by gaboness** [reply]
thank you for your reply and your work for DJL. i am glad to do that，Qwen-72B-Chat model use TikTokenizer，and i find  Qwen1.5-72B-Chat use huggingface tokenizer，finally we choose the latter。
